system/CodeIgniter.php

```diff
@@ -434,6 +435,11 @@
 			{
 				return $possibleRedirect->send();
```

Comment:
While we're here, I don't think there is any need to check for `RedirectResponse` *and* `ResponseInterface` since they are handled the same. Let's remove the first block.